src/database/query-table.c

```diff
@@ -767,6 +811,42 @@ void DB_read_queries(void)
 			upstreamID = findUpstreamID(serv_addr, (in_port_t)serv_port);
 		}
 
+		int reply_type = REPLY_UNKNOWN;
+		if(sqlite3_column_type(stmt, 7) == SQLITE_INTEGER)
+		{
+			// The field has been added for database version 12
+			reply_type = sqlite3_column_int(stmt, 7);
+			if(reply_type < REPLY_UNKNOWN || reply_type >= QUERY_REPLY_MAX)
```

Comment:
According to https://github.com/pi-hole/docs/pull/655
Max reply_type is 13 (a.k.a `blob`). Where does `QUERY_REPLY_MAX` come from?

Reply:
It is the maximum value that is currently supported (so one higher than `BLOB`). This ensures this code stays valid also in the future when new reply types are added.